commodore/config.py

```diff
@@ -85,3 +90,17 @@ def get_configs(self):
 
     def register_config(self, level, repo):
         self._config_repos[level] = repo
+
+    def get_component_aliases(self):
+        return self._component_aliases
+
+    def register_component_aliases(self, aliases: Dict[str, str]):
+        self._component_aliases = aliases
+
+    def verify_component_aliases(self, cluster_parameters: Dict):
+        for alias, cn in self._component_aliases.items():
+            ckey = cn.replace("-", "_")
+            if "_instance" not in cluster_parameters[ckey]:
```

Comment:
Is the idea that `_instance` should just be an empty dict in order to allow instantiation of a component?

I'd propose to do something a little more explicit, like `parameters.<component-name>.multi_instance: true` for example.
Then we could also move the `_instance` magic parameter directly under `parameters` (like in #241 proposed).

Reply:
Changed as suggested in the latest version